test/util/validationUtils.test.js

```diff
@@ -39,7 +39,38 @@ const SUBJECT_AND_ORGANIZATION_QUERY = {
   subject: 'Patient/testPatient',
   measureId: 'testID'
 };
-
+const SUBJECT_AND_PRACTITIONER_QUERY = {
+  periodStart: '2019-01-01',
+  periodEnd: '2019-12-31',
+  status: 'open-gap',
+  practitioner: 'Practitioner/testPractitioner',
+  subject: 'Patient/testPatient',
+  measureId: 'testID'
+};
+const PRACTITIONER_AND_NO_ORG = {
```

Comment:
Sorry didn't catch this before, but isn't this fixture the same as the one above? I'd remove the subject property here and update the test to make sure you're getting the error case you want.